api/frontend.swagger.json

```diff
@@ -141,9 +125,9 @@
         ]
       },
       "delete": {
-        "summary": "Side effects: Any tickets waiting for this backfill will be returned to the active pool, no longer pending.",
+        "summary": "DeleteBackfill receives a backfill ID and deletes its resource.\nAny tickets waiting for this backfill will be returned to the active pool, no longer pending.",
         "description": "BETA FEATURE WARNING:  This call and the associated Request and Response\nmessages are not finalized and still subject to possible change or removal.",
-        "operationId": "FrontendService_DeleteBackfill",
```

Comment:
Not sure, there are so many changes to `swagger.json` files. Do you use a different version of it?